_posts/2020-01-26-celebrate_1k_stars.md

```diff
@@ -0,0 +1,40 @@
+---
+layout: post
+title:  "Celebrating the first 1000 stars on GitHub"
+date:   2020-01-26
+categories: [jump-dev]
+permalink: /blog/1kstars/
+---
+
+JuMP just received its 1000th star on GitHub! We will use this milestone to recap JuMP's growth and other important milestones since its beginnings as an experimental project in 2012.
+
+Tracking the success of some open-source projects can be hard because we might have access to the number of downloads (JuMP's case due to Julia's packge manager implementation), but we can check some metrics like the number of contributors, citations to code or papers describing the project.
+The number of citations of the two main JuMP papers are 253+428 (with some possible double counting).
+Currently 71 developers have added some contribution in some of the 3747 commits of the JuMP repository made available in 67 releases. And this is only data from the main JuMP repo. The ecosystem has many other repos ranging from MOI to solver wrappers.
+
+The number of stars of a GitHub project is yet another indicator, because it means people are adding the project to their list of favorites. However, people that starred JuMP are no necessarily users and many users don't star JuMP (if thats you case give us your star!), therefore its not a precise metric.
```

Comment:
s/no/not/
"if thats you case give us your star!" -> "if that applies to you, consider giving us a star!"